Core/CoreTests/API/APIPostPolicyRequestableTests.swift

```diff
@@ -0,0 +1,60 @@
+//
+// This file is part of Canvas.
+// Copyright (C) 2019-present  Instructure, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+//
+
+import XCTest
+@testable import Core
+
+class PostAssignmentGradesPostPolicyRequestTests: XCTestCase {
+    var req: PostAssignmentGradesPostPolicyRequest!
+
+    override func setUp() {
+        super.setUp()
+        req = PostAssignmentGradesPostPolicyRequest(assignmentID: "1", postPolicy: .everyone)
+    }
+    func testPath() {
+        XCTAssertEqual(req.path, "/api/graphql")
+    }
+
+    func testQuery() {
+        let expected = """
+        mutation PostAssignmentGrades\n    {\n        postAssignmentGrades(input: {assignmentId: \"1\", gradedOnly: false})\n        {\n            assignment { id }\n        }\n    }
+        """
+        XCTAssertEqual(req.query, expected)
+    }
+}
+
+class GetAssignmentPostPolicyInfoRequestTests: XCTestCase {
+    var req: GetAssignmentPostPolicyInfoRequest!
+
+    override func setUp() {
+        super.setUp()
+        req = GetAssignmentPostPolicyInfoRequest(courseID: "1", assignmentID: "1")
+    }
+    func testPath() {
+        XCTAssertEqual(req.path, "/api/graphql")
+    }
+
+    func testQuery() {
+        // swiftlint:disable line_length
+        let expected = """
+        {\n    course(id: \"1\") {\n        sections: sectionsConnection {\n          nodes {\n            id\n            name\n          }\n        }\n      }\n      assignment(id: \"1\") {\n        submissions: submissionsConnection {\n          nodes {\n            score\n            excused\n            state\n            postedAt\n          }\n        }\n      }\n}
```

Comment:
I like that the other test formatted the query with real line breaks. Worth doing that for these other tests too?

Reply:
Not sure.  I copied the text from the test output.  If someone updates the query, and we format, it could still fail b/c of a space or new line which could be difficult to know where that is since these are multi line with spaces and tabs etc.  I think it's just easiest for the sanity of the developer to copy the failing test output and make sure the proper id's are in there.  Just my opinion.

Reply:
That makes sense.. I see the reasoning now.